python/tank_vendor/shotgun_authentication/shotgun_authenticator.py

```diff
@@ -49,7 +77,7 @@ def __init__(self, defaults_manager=None):
         Constructor
 
         :param defaults_manager: A DefaultsManager object that defines the basic
-                                 behaviour of this authenticator. If omitted,
+                                 behavior of this authenticator. If omitted,
```

Comment:
the coding convention to use US english... I sometimes make mistakes because I write UK english by default.. :-)